Pomodoro.Api/ViewModels/Auth/LoginRequestViewModel.cs

```diff
@@ -0,0 +1,28 @@
+﻿// <copyright file="LoginRequestViewModel.cs" company="PomodoroGroup_GL_BaseCamp">
+// Copyright (c) PomodoroGroup_GL_BaseCamp. All rights reserved.
+// </copyright>
+
+using System.ComponentModel.DataAnnotations;
+
+namespace Pomodoro.Api.ViewModels.Auth
+{
+    /// <summary>
+    /// Represent data for login request.
+    /// </summary>
+    public class LoginRequestViewModel
+    {
+        /// <summary>
+        /// Gets or sets email address.
+        /// </summary>
+        [Required(ErrorMessage = "Email is required")]
+        [EmailAddress]
+        public string? Email { get; set; }
+
+        /// <summary>
+        /// Gets or sets password.
+        /// </summary>
+        [Required(ErrorMessage = "Password is required")]
+        [MinLength(8, ErrorMessage = "Password length less than 8 symbols")]
+        public string? Password { get; set; }
```

Comment:
I think it would be better to add the same `MinLength` attribute to the `Password` property in the `RegistrationRequestViewModel`

Reply:
UserManager validate password length when create user, but you right, maybe better check this point during model binding.